faqs/csv_parser/csv_parser.py

```diff
@@ -0,0 +1,85 @@
+import csv
+
+
+class CsvContentParser:
+    filename = ""
+    locale = ""
+    topic_map = {}
+    current_topic = ""
+    delimiter = ''
```

Comment:
We probably don't want to define these variable at the class level. `filename`, `locale` and `delimiter` are all set in `__init__`. Defining `topic_map` here is especially dangerous because it's a dict, and thus mutable. As the code is now, a single `topic_map` dict is shared between all instances of this class (which is likely not what users of the class want).

Since each `RowEntry` knows its topic, we can probably just do away with `topic_map` entirely, and only keep track of `current_topic` in `parse` (we'll need to pass it back and forth from `handle_row` probably, or adjust how `handle_row` works a bit).